src/components/FermentationRecipeList/FermentationRecipe.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * @typedef {import("../../index").farmhand.item} item
+ * @typedef {import("../../index").farmhand.keg} keg
+ */
+
+import React, { useContext, useEffect, useState } from 'react'
+import { object } from 'prop-types'
+import Card from '@material-ui/core/Card'
+import CardHeader from '@material-ui/core/CardHeader'
+import CardActions from '@material-ui/core/CardActions'
+import Button from '@material-ui/core/Button'
+
+import { PURCHASEABLE_CELLARS } from '../../constants'
+import { items } from '../../img'
+import { doesCellarSpaceRemain } from '../../utils/doesCellarSpaceRemain'
+import { getMaxYieldOfFermentationRecipe } from '../../utils/getMaxYieldOfFermentationRecipe'
+import { getSaltRequirementsForFermentationRecipe } from '../../utils/getSaltRequirementsForFermentationRecipe'
+import { FERMENTED_CROP_NAME } from '../../templates'
+import QuantityInput from '../QuantityInput'
+import FarmhandContext from '../Farmhand/Farmhand.context'
+import { fermentableItemsMap, itemsMap } from '../../data/maps'
+
+import './FermentationRecipe.sass'
+import { getInventoryQuantityMap } from '../../utils/getInventoryQuantityMap'
+import { integerString } from '../../utils'
+import AnimatedNumber from '../AnimatedNumber'
+import { memoize } from '../../utils/memoize'
+
+/**
+ * @type {function(keg[], item):number}
+ */
+const getRecipesInstancesInCellar = memoize(
+  /**
+   * @param {keg[]} cellarInventory
+   * @param {item} item
+   * @returns number
+   */
+  (cellarInventory, item) => {
+    return cellarInventory.reduce((acc, keg) => {
+      if (keg.itemId === item.id) {
+        acc++
+      }
+
+      return acc
+    }, 0)
```

Comment:
another way to consider writing this which i think is a bit more concise, but ultimately the result is the same would be:

```
return cellarInventory.filter((keg) => keg.itemId === itemId).length
```